draft-ietf-ppm-dap.md

```diff
@@ -2731,90 +2611,65 @@ chosen independently of the generated reports.
-each bit of the measurement value.
+tasks increases superlinearly as a function of the chosen bit-length, as
+multiple rounds of evaluation are needed for each bit of the measurement value,
+and each bit of the measurement value requires computation which scales with the
+chosen bit-length.
 
 When dealing with variable length measurements (e.g domain names), it is
-necessary to pad them to convert into fixed-size measurements. When computing
-the heavy hitters from a batch of such measurements, we can early-abort the
-Poplar1 execution once we have reached the padding region for a candidate
+necessary to pad them to convert into fixed-length measurements. When computing
+the heavy hitters from a batch of such measurements, VDAF execution can be
+terminated early once once the padding region is reached for a given
 measurement. For smaller length measurements, this significantly reduces the
 cost of communication between Aggregators and the steps required for the
 computation. However, malicious Clients can still generate maximum length
 measurements forcing the system to always operate at worst-case performance.
 
-[[TODO: Revisit this paragraph once https://github.com/cfrg/draft-irtf-cfrg-vdaf/issues/273
-is resolved.]]
+[[TODO: Revisit this paragraph once
+https://github.com/cfrg/draft-irtf-cfrg-vdaf/issues/273 is resolved.]]
 
-Therefore, care must be taken that a DAP deployment can comfortably handle
-computation of measurements for arbitrarily large sizes, otherwise, it may
-result in a DoS possibility for the entire system.
+Therefore, care must be taken that a DAP deployment can handle VDAF execution of
+all possible measurement values for any tasks that the deployment is configured
+for. Otherwise, an attacker may deny service by uploading many expensive
+reports.
 
-## Differential privacy {#dp}
+### Task configuration agreement and consistency
 
-Optionally, DAP deployments can choose to ensure their aggregate results achieve
-differential privacy [Vad16]. A simple approach would require the Aggregators to
-add two-sided noise (e.g. sampled from a two-sided geometric distribution) to
-aggregate shares. Since each Aggregator is adding noise independently, privacy
-can be guaranteed even if all but one of the Aggregators is malicious.
-Differential privacy is a strong privacy definition, and protects users in
-extreme circumstances: even if an adversary has prior knowledge of every
-measurement in a batch except for one, that one record is still formally
-protected.
+In order to execute a DAP task, it is necessary for all parties to ensure they
+agree on the configuration of the task. However, it is possible for a party to
+participate in the execution of DAP without knowing all of the task's
+parameters. For example, a Client can upload a report ({{upload-flow}}) without
+knowing the minimum batch size that is enforced by the Aggregators during
+collection ({{collect-flow}}).
 
-## Robustness in the presence of malicious servers
```

Comment:
I removed this section as the relevant issue is closed, and also because we now specify to two aggregators so aggregator subset selection no longer makes sense. (I think something like this idea could work if we had 3+ independent DAP aggregator deployments, sent each report to all deployments, and collected from all of them -- but IMO that's outside of the scope of what needs to be mentioned in the DAP spec.)